server/util/testing/flags/flags.go

```diff
@@ -7,10 +7,21 @@ import (
 	"testing"
 
 	"github.com/buildbuddy-io/buildbuddy/server/config"
+	"github.com/buildbuddy-io/buildbuddy/server/environment"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+// Set a flag value and register a cleanup function to restore the flag
+// to its original value after the given test is complete.
+func SetAndReconcile(t testing.TB, name, value string, env environment.Env) {
```

Comment:
nit: add a TODO that this will be removed in final cleanup?

Reply:
```suggestion
// TODO: Remove this function when we no longer need to reconcile flags
func SetAndReconcile(t testing.TB, name, value string, env environment.Env) {
```